gtsam/gtsam.i

```diff
@@ -1684,7 +1695,8 @@ class GaussianFactorGraph {
   GaussianFactorGraph(const gtsam::GaussianBayesTree& bayesTree);
 
   // From FactorGraph
-  void print(string s="") const;
+  void print(string s = "FactorGraph",
```

Comment:
Leave this empty or change it to `GaussianFactorGraph`?

Reply:
Should be same default as c++ which should not change @varunagrawal so change before merge if you *did* change.

Reply:
This is the default for the base class, so it will also print in the subclass. Should I override the print method?

Reply:
To clarify, the `print` in `FactorGraph` has "FactorGraph" as the default and since `GaussianFactorGraph` inherits from `FactorGraph`, it also prints "FactorGraph" as the default (which to me isn't quite right).

Reply:
Not this PR to fix it